data/json/items/magazine/40mm.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "id": "belt40mm",
```

Comment:
The belt itself is a good idea but does the encumbrance calculation code I wrote produce sensible values for a full belt of this?